docs/reference/docs/get.asciidoc

```diff
@@ -228,5 +228,7 @@ it's current version is equal to the specified one. This behavior is the same
 for all version types with the exception of version type `FORCE` which always
 retrieves the document.
 
-Note that Elasticsearch do not store older versions of documents. Only the current version can be retrieved.
+Note that even though Elasticsearch does not delete versions of documents in
+an index to ensure constant sequential writes, older versions of documents are
+unreachable. Only the current version can be retrieved.
```

Comment:
I find this sentence a bit confusing. We do delete the old document on update, it's just that the disk space in the lucene index is not claimed till a merge happens, given that segments are immutable. In fact deleted documents are only marked as deleted and purged later on. I think you were referring to this implementatio detail right? Might be me, but this sentence makes me think that elasticsearch does keep old versions around but doesn't allow users to retrieve them. Makes sense?